modules/bazel_bats/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://github.com/filmil/bazel-bats",
+    "maintainers": [
+        {
+            "email": "filmil@gmail.com",
+            "github": "filmil",
+            "name": "Filip Filmar"
+        }
+    ],
+    "repository": [],
```

Comment:
Please add `github:filmil/bazel-bats` as the source repository

Reply:
Done.